Kitodo/src/main/java/de/sub/goobi/helper/GoobiScript.java

```diff
@@ -698,6 +703,7 @@ public void deleteTiffHeaderFile(List<Process> processes) {
                 }
                 Helper.setMeldung(KITODO_SCRIPT_FIELD, "TiffHeaderFile deleted: ", process.getTitle());
             } catch (Exception e) {
+                logger.error(e.getMessage(), e);
```

Comment:
Please replace this log call and following `Helper.setFehlermeldung` with `Helper.setErrorMessage` which bundles both calls.